src/cluster-legacy.h

```diff
@@ -0,0 +1,360 @@
+#ifndef CLUSTER_LEGACY_H
```

Comment:
I think we normally use _ for naming of the .h and .c files.

Reply:
done